components/01-atoms/links/link.twig

```diff
@@ -0,0 +1,25 @@
+{#
+ # Available variables:
+ # - link_content - the content of the link (typically text)
+ # - link_url - the url this link should poing to
+ # - link_attributes - array of attribute,value pairs
+ #
+ # Available blocks:
+ # - link_content - used to replace the content of the link
+ #     Example: to insert the image component
+ #}
+
+{% set link_base_class = link_base_class|default('link') %}
+
+{# If link_attributes is not defined, set it to an empty object by default #}
+{% set link_attributes = link_attributes|default({}) %}
+
+{% set link_attributes = link_attributes|merge({
+  class: bem(link_base_class, link_modifiers, link_blockname)
+}) %}
+
```

Comment:
I think this would be cleaner but correct me if I'm wrong 
```
{% set link_attributes = link_attributes|merge({
  class: bem(link_base_class, link_modifiers, link_blockname),
  href: link_url
}) %}
```

Reply:
Ooh, yeah! I like that

Reply:
Done. Totes cleaner!